ui/src/modules/plans/models/operation/operation-info.models.ts

```diff
@@ -33,137 +33,133 @@ import {
     OperationSort,
     OperationWrite,
 } from './type'
+import NodeStyles = SgNodeControl.NodeStyles
```

Comment:
@TerranceKhumalo-absa it would be great to not remove the namespace, but replace it with a module that could provide some services/providers in it.